kafka-plugins-0.10/pom.xml

```diff
@@ -19,6 +19,16 @@
       <artifactId>kafka-plugins-common</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.plugin</groupId>
+      <artifactId>format-common</artifactId>
+      <version>${cdap.plugin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.cdap.plugin</groupId>
+      <artifactId>format-avro</artifactId>
```

Comment:
why does it need format-avro? If avro is missing we should add it directly and not indirectly through this.

Reply:
fixed